cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final Locale DE = Locale.forLanguageTag("de");
+    private static final Locale EN_US = Locale.forLanguageTag("en-US");
+    private static final List<Locale> SUPPORTED_LOCALES = asList(DE, EN_US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
+    private static final String TOKEN = "b4c0n73n7fu1";
+    private ContentfulCmsService contentfulCmsService;
+
+    @Before
+    public void setUp() throws Exception {
+        contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
+    }
+
+    @Test
+    public void whenAskForExistingStringContentThenGet() throws Exception {
+        CmsIdentifier identifier = CmsIdentifier.ofEntryTypeAndKeyAndField("human", "Finn", "description");
+        CompletionStage<Optional<String>> optionalCompletionStage = contentfulCmsService.get(SUPPORTED_LOCALES, identifier);
+        Optional<String> content = optionalCompletionStage.toCompletableFuture().get(5, TimeUnit.SECONDS);
```

Comment:
I was trying to :
https://github.com/commercetools/commercetools-sunrise-java-cms/pull/2/commits/34a6ab4f66a4a75d3e74feb4aa614521ff455797
but build failed, so I tried this:
https://github.com/commercetools/commercetools-sunrise-java-cms/pull/2/commits/3258726d4cea6db85f6ea418cf2a10465c4de7fb
which is still wrong.
I created this method in 2 test classes as private methods:
https://github.com/commercetools/commercetools-sunrise-java-cms/pull/2/commits/b30f51f65c543efc7086533ae13d07849a7a56a3
I wonder if modying build.sbt wouldn't be a better idea, so I could have TestHelper class, which would be shared along test and it test. But then I will have some extra folder, it could be ugly...